README.md

```diff
@@ -0,0 +1,73 @@
+<p align="right">
+  <img  src="./src/odemis/gui/img/logo_h30.png">
+</p>
+
+# Odemis
+*Open Delmic Microscope Software*
+
+## Requirements
+* Linux (tested on Ubuntu 16.04 and 18.04 x86 32-bits and 64-bits)
+* Python (v3.5)

```

Comment:
Isn't python 3.6 the preferred version since this is the standard in Ubuntu 18?

Reply:
Well, we still support Python 3.5 (and 2.7!). So that's good enough. I guess that ideally it would indicate all the Python versions supported.
This readme file could be a lot further improved, but that's already a good step :-)